objectivec/GPBDescriptor.m

```diff
@@ -819,13 +819,15 @@ - (void)calcValueNameOffsets {
       return;
     }
     uint32_t *offsets = malloc(valueCount_ * sizeof(uint32_t));
-    const char *scan = valueNames_;
-    for (uint32_t i = 0; i < valueCount_; ++i) {
-      offsets[i] = (uint32_t)(scan - valueNames_);
-      while (*scan != '\0') ++scan;
-      ++scan;  // Step over the null.
+    if (offsets) {
```

Comment:
Use an early exit here.

I almost wonder if we should do an `NSAssert` in `DEBUG` builds also, but I guess it isn't worth it since it really shouldn't happen.

Reply:
Yeah, it is not happening in real scenarios. This is just to silent Static Analyzer messages.